internal/stringset.go

```diff
@@ -18,3 +20,15 @@ func (s Set) Contains(i string) bool {
 	_, ok := s[i]
 	return ok
 }
+
+// ToSlice returns a sorted slice of strings that are contained within the set.
+func (s Set) ToSlice() []string {
```

Comment:
![khan](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia1.tenor.com%2Fimages%2Fd7e36d0aacab6af7dd9cab425d23c31a%2Ftenor.gif%3Fitemid%3D5575846&f=1&nofb=1)

Reply:
?

Reply:
Just referring to our discussion/frustration with go's lack of native support for sets and the impetus for rolling your own 😃